### DISCUSSION TITLE
Add quick find navigation by typing first letters without activating search box

### DISCUSSION THREAD

**Discussion by alex1yaremchuk** [doublecmd/doublecmd#1918] [Suggestions]
I suggest to bring tc-like files navigation by first letters of file/dir names without brining up a find dialog. 

It it extremely inconvenient for me to have focus change to find dialog. 

sometimes i don't understand why cursor doesn't respond to direction keys - and it turns out find dialog is focused - what a mess! 

Please consider reproducing this tc functionality names Quick Search - Letter Only.

**Comment by breaker27** [comment]
What exactly is the problem when the find dialog has the active focus?

For me (Linux, DC 1.1.19~585), cursor up/down react totally normal to go to previous/next file in the list even the find dialog has the cursor focus. I didn't see any problem until now and use the quick filter function regularly.

**Reply by alex1yaremchuk** [reply]
> What exactly is the problem when the find dialog has the active focus?
> 
> For me (Linux, DC 1.1.19~585), cursor up/down react totally normal to go to previous/next file in the list even the find dialog has the cursor focus. I didn't see any problem until now and use the quick filter function regularly.

Right, you can move between filtered items, but it is not possible to use right arrow to drill into the selected directory. 
Enter is working, but it is a bit disruptive for this routine. 

Hitting escape feels is a bit to much for this workflow. 
That's why TC always had this jumping the files with first letters seamlessly without entering additional dialogs on the way.

**Reply by alex1yaremchuk** [reply]
And sometimes I don't notice this dialog is active and I can't get why tool is not responding properly to my commands.
